docs/streams/developer-guide/config-streams.html

```diff
@@ -259,7 +259,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
           </tr>
           <tr class="row-even"><td>cache.max.bytes.buffering</td>
             <td>Medium</td>
-            <td colspan="2">Maximum number of memory bytes to be used for record caches across all threads.</td>
+            <td colspan="2">Maximum number of memory bytes to be used for record caches across all threads. Deprecated.</td>
```

Comment:
Should we add "Deprecated" to the "head line" in L260 instead? (Similar below)

Reply:
yes! done